lib/ansible/parsing/utils/yaml.py

```diff
@@ -36,9 +36,10 @@ def _handle_error(yaml_exc, file_name, show_content):
         err_obj = AnsibleBaseYAMLObject()
         err_obj.ansible_pos = (file_name, yaml_exc.problem_mark.line + 1, yaml_exc.problem_mark.column + 1)
 
-    err_msg = getattr(yaml_exc, 'problem', '')
+    err_msg = 'We were unable to read either as JSON nor YAML, these are the errors we got from each:\n' \
+              'JSON: %s\n\n' % to_text(json_exc) + YAML_SYNTAX_ERROR % getattr(yaml_exc, 'problem', '')
```

Comment:
```suggestion
              'Syntax Error while loading JSON.\n  %s\n\n' % to_text(json_exc) + YAML_SYNTAX_ERROR % getattr(yaml_exc, 'problem', '')
```

Reply:
Maybe make the JSON error message match `YAML_SYNTAX_ERROR`. That way the output is more uniform:
```
ERROR! Unable to load as JSON or YAML.

Syntax Error while loading JSON:
  Expecting value: line 1 column 1 (char 0)

Syntax Error while loading YAML.
  mapping values are not allowed in this context
...
```